src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -686,6 +760,67 @@ def cmac_algorithm_supported(self, algorithm):
     def create_cmac_ctx(self, algorithm):
         return _CMACContext(self, algorithm)
 
+    def sign_x509_request(self, builder, private_key, algorithm):
+        # TODO: check type of private key parameter.
```

Comment:
Renamed to `create_x509_csr()` and added this to the X.509 backend interface too.